src/main/java/com/facebook/openwifirrm/DeviceConfig.java

```diff
@@ -30,6 +30,9 @@ public class DeviceConfig {
 	/** The RRM schedule (if enabled), only applied at the zone or network layer */
 	public RRMSchedule schedule;
 
+	/** earameters for the algorithms, mapping algorithm name to string of a list of parameters */
+	public Map<String, String> algorithmsParameters;
+
```

Comment:
This is already defined in the `RRMSchedule` class, don't duplicate it here: https://github.com/Telecominfraproject/wlan-cloud-rrm/blob/6fb15f777cc8034e9e67fd6bdfd0e75f5aeeac89/src/main/java/com/facebook/openwifirrm/RRMSchedule.java#L31

Reply:
ah missed that, thanks for the catch!